syntax = "proto3";
option optimize_for = LITE_RUNTIME;
package ii.proto;

message InputFrame {
  sint64 velocity_x = 1;
  sint64 velocity_y = 2;

  bool target_relative = 3;
  sint64 target_x = 4;
  sint64 target_y = 5;

  uint32 keys = 6;
}

